### PR TITLE
Add consul alerts

### PR DESCRIPTION
**Pull request by offlinehacker**

### CODE REVIEW

File: pkgs/build-support/fetchzip/default.nix [hunk 1]
```diff
@@ -28,7 +28,7 @@ lib.overrideDerivation (fetchurl ({
       mkdir "$unpackDir"
       cd "$unpackDir"
 
-      renamed="$TMPDIR/$name"
+      renamed="$TMPDIR/${baseNameOf url}"
```
Comment (copumpkin) [inline pkgs/build-support/fetchzip/default.nix]: Does this change belong in this PR? It seems like it'll cause quite a few things to rebuild.

Comment (offlinehacker) [inline pkgs/build-support/fetchzip/default.nix]: Noup, it does not, i will rebase this and it will go away.

**Comment by offlinehacker** [comment]
This should be using new buildGoPackage

**Comment by offlinehacker** [comment]
I will use fix it when https://github.com/NixOS/nixpkgs/pull/6119 will be merged.

**Comment by offlinehacker** [comment]
Ok, it's now using buildGoPackage, i'm merging it.

**Pull request merged**